CMake error with Google Tests

As reported by @alainmarcel:
```bash
alain@alain-MacBook:~/Surelog$ make
mkdir -p build/tests;
mkdir -p build/dist;
mkdir -p dist;
cd build; cmake ../ -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local
-- Building without demo. To enable demo build use: -DWITH_DEMO=True
-- Output libraries to /home/alain/Surelog/dist
CMake Error at third_party/googletest/CMakeLists.txt:13 (if):
  if given arguments:

    "CMAKE_VERSION" "VERSION_GREATER_EQUAL" "3.1"

  Unknown arguments specified


-- Configuring incomplete, errors occurred!
See also "/home/alain/Surelog/build/CMakeFiles/CMakeOutput.log".
Makefile:6: recipe for target 'release' failed
make: *** [release] Error 1
```

Seems to be tracked here: https://github.com/google/googletest/issues/2711.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -198,7 +198,24 @@ file(
   ${PROJECT_SOURCE_DIR}/src/SourceCompile/CompilationUnit.cpp
   ${PROJECT_SOURCE_DIR}/src/SourceCompile/ParseFile.cpp
   ${PROJECT_SOURCE_DIR}/src/SourceCompile/SV3_1aTreeShapeListener.cpp
-  ${PROJECT_SOURCE_DIR}/src/DesignCompile/*.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/Builtin.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileClass.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileDesign.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileFileContent.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileHelper.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileModule.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompilePackage.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileProgram.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileStep.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/CompileToolbox.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/DesignElaboration.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/ElaborationStep.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/NetlistElaboration.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/PackageAndRootElaboration.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/ResolveSymbols.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/TestbenchElaboration.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/UhdmWriter.cpp
+  ${PROJECT_SOURCE_DIR}/src/DesignCompile/UVMElaboration.cpp
   ${PROJECT_SOURCE_DIR}/src/Testbench/*.cpp
   ${PROJECT_SOURCE_DIR}/src/CommandLine/*.cpp
   ${PROJECT_SOURCE_DIR}/src/Common/*.cpp
@@ -208,9 +225,6 @@ file(
   ${PROJECT_SOURCE_DIR}/src/Config/*.cpp
   ${PROJECT_SOURCE_DIR}/src/Expression/*.cpp
   ${PROJECT_SOURCE_DIR}/src/Package/*.cpp)
-
-# Disabling Google tests until build issues are ironed out
-#list(FILTER surelog_SRC EXCLUDE REGEX ".*_test.cpp$")
 
 set(SURELOG_PUBLIC_HEADERS ${PROJECT_SOURCE_DIR}/src/surelog.h)
 

--- a/src/DesignCompile/CompileHelper.h
+++ b/src/DesignCompile/CompileHelper.h
@@ -32,11 +32,12 @@
 #include "Expression/ExprBuilder.h"
 #include "ErrorReporting/ErrorContainer.h"
 #include "Common/PortNetHolder.h"
-#include "uhdm.h"
 
 namespace UHDM { 
  class assignment;
  class tf_call;
+ class BaseClass;
+ typedef std::vector<BaseClass*> VectorOfany;
 };
 
 namespace SURELOG {

